manifests/config/global/framework.pp

```diff
@@ -4,20 +4,46 @@
 
 # == Class rundeck::config::global::framework
 #
-# This private class is called from rundeck::config used to manage the framework properties of rundeck
+# This private class is called from rundeck::config and
+# used to manage the framework properties of rundeck
 #
 class rundeck::config::global::framework(
-  $properties_dir   = $rundeck::config::properties_dir,
-  $user             = $rundeck::config::user,
-  $group            = $rundeck::config::group
-
+  $properties_dir = $rundeck::config::properties_dir,
+  $user           = $rundeck::config::user,
+  $group          = $rundeck::config::group,
 ) {
 
   $properties_file = "${properties_dir}/framework.properties"
 
-  ensure_resource('file', $properties_dir, {'ensure' => 'directory', 'owner' => $user, 'group' => $group } )
+  ensure_resource('file', $properties_dir, {
```

Comment:
same as here, what's the point to change it, with your personal style? If it is, then change in seperate pull request. Don't mix it.